src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -721,37 +721,34 @@ InferenceProfiler::Measure(
   std::map<cb::ModelIdentifier, cb::ModelStatistics> end_status;
   cb::InferStat start_stat;
   cb::InferStat end_stat;
-  long long measurement_window_ms;
 
   if (include_server_stats_) {
     RETURN_IF_ERROR(GetServerSideStatus(&start_status));
   }
   RETURN_IF_ERROR(manager_->GetAccumulatedClientStat(&start_stat));
 
+  std::chrono::nanoseconds window_start_ns =
```

Comment:
Instead of capturing a new start on each run, do you think we should capture start only on the first run? For the later trials, the start will be the end of the previous trial. This ensures that the measurement windows do not have any gaps between them.

Reply:
Changed to have start time be previous window end time 👍